docs/appendices/comparison-to-related-tools.md

```diff
@@ -138,6 +138,26 @@ there are some major limitations:
         not every CloudTrail action matches IAM Actions, the results are
         not always accurate.
 
+Monitoring AWS API Calls
+------------------------
+
+### actionhero
+
+-   [actionhero](https://github.com/princespaghetti/actionhero)
+
+Action Hero is a sidecar style utility to monitor the AWS API calls that your
+application makes and log out the corresponding IAM actions.
+
+Action Hero is limited in that it cannot capture parameters and does not
```

Comment:
Nit: It doesn't capture parameters *or* resource ARNs (due to limitations from AWS's Client-Side Monitoring)

Reply:
By "parameters" I think they meant ARNs too. But yea last night I thought it would be good to be more specific about that too

Reply:
included resource ARNs in wording